scripts/workers/remote_submission_worker.py

```diff
@@ -576,6 +580,15 @@ def run_submission(
     else:
         status = "failed"
         submission_data["submission_status"] = status
+
+    if challenge_phase.get("challenge").get("is_docker_based"):
+        sender_email = settings.CLOUDCV_TEAM_EMAIL
+        user_email = submission.get("created_by").get("email")
+        template_id = settings.SENDGRID_SETTINGS.get("TEMPLATES").get(
+            "TASK_DONE_NOTIFICATION"
+        )
+        send_email(sender_email, user_email, template_id)
+
```

Comment:
`remote_submission_worker.py` will not be evaluating submissions for docker/code upload based challenge.

Reply:
Oh, I didn't know that. So `rl_submission_worker.py` evals docker-based submissions?

Reply:
Instead of adding in workers, can you add here: `jobs/views.py: update_submission`?
The workers will return result to this endpoint and the success from this endpoint means that the submission is successfully evaluated!

Reply:
I also started to think it is better to make changes in `update_submission`. Thanks for your kind guidance!